### DISCUSSION TITLE
Apply ui.colors to all pages

### DISCUSSION THREAD

**Discussion by ghbm-itk** [zauberzeug/nicegui#3787] [Ideas / Feature Requests]
### Description

I would like to call ui.colors(primary="#cc0000") just before ui.run() to set my color theme.
It seems that I have to recall it every time a new page is loaded by ui.page.

```python
from nicegui import ui

@ui.page("/colors")
def colors():
    ui.label("This page has colors")
    ui.colors(primary="#ff0000")
    ui.button("No colors", on_click=lambda: ui.navigate.to("/no_colors"))
    

@ui.page("/no_colors")
def no_colors():
    ui.label("This page has no colors")
    ui.button("Colors", on_click=lambda: ui.navigate.to("/colors"))

ui.colors(primary="#ff0000")
ui.run()
ui.navigate.to("/colors")
```

**Comment by falkoschindler** [comment]
Hi @ghbm-itk,

This is the way `ui.colors` works. Like other `ui.` elements, it customizes the current page. Have a look into [modularization example](https://github.com/zauberzeug/nicegui/tree/main/examples/modularization) for how to re-use page customizations like `ui.headers` or `ui.colors` on multiple pages.

I converted this issue into a feature request, because something like `app.colors` or a more generic API for defining app-wide layout and styling might be worth thinking about.

**Comment by falkoschindler** [comment]
This feature has been implemented in PR #5611 for version 3.6. ✔️
